pytorch_lightning/loggers/wandb.py

```diff
@@ -468,6 +496,45 @@ def after_save_checkpoint(self, checkpoint_callback: "ReferenceType[ModelCheckpo
         elif self._log_model is True:
             self._checkpoint_callback = checkpoint_callback
 
+    @classmethod
+    @rank_zero_only
+    def download_artifact(
+        self, artifact: str, dir: Optional[str] = None, type: Optional[str] = None, use_artifact: Optional[bool] = True
```

Comment:
This is a classmethod convention suggests:

```suggestion
        cls, artifact: str, dir: Optional[str] = None, type: Optional[str] = None, use_artifact: Optional[bool] = True
```